kernel/indTyping.ml

```diff
@@ -96,10 +96,6 @@ let check_context_univs ~ctor env info ctx =
   in
   fst (Context.Rel.fold_outside ~init:(info,env) check_one ctx)
 
-let check_indices_matter env_params info indices =
-  if not (indices_matter env_params) then info
-  else check_context_univs ~ctor:false env_params info indices
```

Comment:
Removing this is more like always having indices not mattering.
If you try to make indices matter the definition of `eq` is squashed, so eq_rect and company don't exist and nothing compiles.